postfix_exporter.go

```diff
@@ -111,6 +111,11 @@ func CollectTextualShowqFromReader(file io.Reader, ch chan<- prometheus.Metric)
 		},
 		[]string{"queue"})
 
+	for _, q := range []string{"active", "hold", "other"} {
```

Comment:
Be sure to add a comment to these blocks to indicate why this is being done... :-)